src/app/features/settings-dropdown/settings-dropdown.tsx

```diff
@@ -101,6 +102,21 @@ export function SettingsDropdown() {
             >
               Change theme
             </MenuItem>
+            <MenuItem
+              data-testid={SettingsSelectors.ToggleTheme}
+              onClick={wrappedCloseCallback(() => {
+                void analytics.track('get_support_menu_item');
```

Comment:
Per convention should be a verb @markmhx `clicked_get_support_menu_item`?

Reply:
![image](https://user-images.githubusercontent.com/28991/203126204-2b325aae-691d-42c9-95b5-a30a9a7b05d1.png)